### PR TITLE
Adding call for sponsorship page

### PR DESCRIPTION
**Pull request by supriyog**
There are 3 changes:
1. Adding a new page for "call for sponsorship" within _posts/2021-12-10-sponsor-cfp.html
2. Updated _includes/header.html to include "call for sponsorship" in nav bar
3. Updated organisation for Supriyo Ghosh within committee page.

### CODE REVIEW

File: _posts/2021-11-19-committee.html
Comment (haz) [inline _posts/2021-11-19-committee.html:118]: @supriyog Hehe, congrats!

**Review by haz — APPROVED** [review]
Looks good to me. Just waiting on approval from the sponsor chairs before deploying (will happen upon merge).

File: _posts/2021-11-19-committee.html
Comment (supriyog) [reply _posts/2021-11-19-committee.html:118]: Thanks a lot @haz :D

**Pull request merged**
